<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="614" started-at="2018-10-16T07:29:31Z" finished-at="2018-10-16T07:29:32Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="614" started-at="2018-10-16T07:29:31Z" finished-at="2018-10-16T07:29:32Z">
      <class name="com.test.ExtentReportTest">
        <test-method status="PASS" signature="startReport()[pri:0, instance:com.test.ExtentReportTest@28feb3fa]" name="startReport" is-config="true" duration-ms="204" started-at="2018-10-16T12:59:31Z" finished-at="2018-10-16T12:59:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method status="PASS" signature="Register()[pri:0, instance:com.test.ExtentReportTest@28feb3fa]" name="Register" duration-ms="11" started-at="2018-10-16T12:59:31Z" finished-at="2018-10-16T12:59:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Register -->
        <test-method status="PASS" signature="generateResults(org.testng.ITestResult)[pri:0, instance:com.test.ExtentReportTest@28feb3fa]" name="generateResults" is-config="true" duration-ms="4" started-at="2018-10-16T12:59:31Z" finished-at="2018-10-16T12:59:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Register status=SUCCESS method=ExtentReportTest.Register()[pri:0, instance:com.test.ExtentReportTest@28feb3fa] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateResults -->
        <test-method status="FAIL" signature="login()[pri:0, instance:com.test.ExtentReportTest@28feb3fa]" name="login" duration-ms="3" started-at="2018-10-16T12:59:31Z" finished-at="2018-10-16T12:59:31Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.test.ExtentReportTest.login(ExtentReportTest.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="generateResults(org.testng.ITestResult)[pri:0, instance:com.test.ExtentReportTest@28feb3fa]" name="generateResults" is-config="true" duration-ms="1" started-at="2018-10-16T12:59:31Z" finished-at="2018-10-16T12:59:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=login status=FAILURE method=ExtentReportTest.login()[pri:0, instance:com.test.ExtentReportTest@28feb3fa] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateResults -->
        <test-method status="SKIP" signature="mobileRecharge()[pri:0, instance:com.test.ExtentReportTest@28feb3fa]" name="mobileRecharge" duration-ms="4" started-at="2018-10-16T12:59:31Z" finished-at="2018-10-16T12:59:31Z">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[This is skipped, not  ready for testing]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: This is skipped, not  ready for testing
	at com.test.ExtentReportTest.mobileRecharge(ExtentReportTest.java:60)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- mobileRecharge -->
        <test-method status="PASS" signature="generateResults(org.testng.ITestResult)[pri:0, instance:com.test.ExtentReportTest@28feb3fa]" name="generateResults" is-config="true" duration-ms="1" started-at="2018-10-16T12:59:31Z" finished-at="2018-10-16T12:59:31Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=mobileRecharge status=SKIP method=ExtentReportTest.mobileRecharge()[pri:0, instance:com.test.ExtentReportTest@28feb3fa] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- generateResults -->
        <test-method status="PASS" signature="close()[pri:0, instance:com.test.ExtentReportTest@28feb3fa]" name="close" is-config="true" duration-ms="365" started-at="2018-10-16T12:59:31Z" finished-at="2018-10-16T12:59:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- com.test.ExtentReportTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
